src/test/calcLogic.test.js

```diff
@@ -0,0 +1,25 @@
+import calculate from '../logic/calculate';
+import operate from '../logic/operate';
+
+describe('calculate testing', () => {
+  it('should return an object with null values for total, next, and operation when buttonName is "AC"', () => {
+    const result = calculate({}, 'AC');
+    expect(result).toEqual({ total: null, next: null, operation: null });
+  });
+
+  it('should perform calculation when buttonName is "="', () => {
+    const result = calculate({ total: '2', next: '3', operation: '+' }, '=');
+    expect(result).toEqual({ total: '5', next: null, operation: null });
+  });
+});
+describe('operate testing', () => {
+  it('should return a sum if "+" is assigned between two numbers', () => {
+    const result = operate(1, 5, '+');
+    expect(result).toEqual('6');
+  });
+
+  it('should return a multiplication if "x" is assigned between two numbers', () => {
+    const result = operate(10, 3, 'x');
+    expect(result).toEqual('30');
+  });
+});
```

Comment:
- [ ] @mnaibei and @RHK-MICROVERSE great work on your tests, however, you didn't cover all functionalities and scenarios while testing the `operate.js` and `calculate.js` modules.

For instance, in the calculate.js file the if statement shown below represents a scenario, the same case applies to the operate.js file
![image](https://user-images.githubusercontent.com/61408860/235878460-b200022f-b53b-448f-b995-f25ef0dcef09.png)

- [ ] @team While you did a great job testing the components you forgot to add tests for Navbar and Display components, kindly consider adding tests for these components as it was part of the project requirement as shwon below

![image](https://user-images.githubusercontent.com/61408860/235879278-d9d8af74-8e1a-4cb1-894c-51cedbb6ac94.png)